WordPress/Classes/ViewRelated/Notifications/Tools/NotificationCommentDetailCoordinator.swift

```diff
@@ -38,8 +40,9 @@ class NotificationCommentDetailCoordinator: NSObject {
         configure(with: notification)
 
         if let comment = loadCommentFromCache(commentID) {
-            createViewController(comment: comment)
-            completion(viewController)
+            createViewController(comment: comment, completion: {
+                completion(self.viewController)
+            })
```

Comment:
Optional stylistic suggestion:

```suggestion
            createViewController(comment: comment) {
                completion(self.viewController)
            }
```

Reply:
Oh, yea. Sometimes I forget you can do that. Thanks!